Lib/test/test_warnings/__init__.py

```diff
@@ -125,6 +125,8 @@ def test_ignore(self):
             self.module.filterwarnings("ignore", category=UserWarning)
             self.module.warn("FilterTests.test_ignore", UserWarning)
             self.assertEqual(len(w), 0)
+            self.assertEqual(list(__warningregistry__.keys()),
```

Comment:
This line doesn't look too long for splitting it. Especially if remove redundant `.keys()`.